src/components/pages/get-involved/form.js

```diff
@@ -10,18 +10,29 @@ const GetInvolvedForm = () => (
     target="_blank"
     noValidate
   >
-    <Row>
-      <div className={formStyle.grid}>
-        <Input
-          label="Email address"
-          type="email"
-          name="EMAIL"
-          id="action-form-email"
-          isRequired
-        />
-      </div>
-    </Row>
-
+    <Col width={[4, 6, 6]} padding={[0, 0, 0]} className={formStyle.grid}>
```

Comment:
Could you please center this on Desktop?
![image](https://user-images.githubusercontent.com/18607205/101204545-9d90be80-3629-11eb-99cd-68fe3fd9a821.png)

Reply:
Do you mean to extend the input bars to the width of the column like this?
<img width="1680" alt="Screen Shot 2020-12-05 at 8 31 24 PM" src="https://user-images.githubusercontent.com/55467415/101269182-db7d0800-3739-11eb-8fcb-fe20133aa0eb.png">

Presently it is centered like the other content on the website.
![combo](https://user-images.githubusercontent.com/55467415/101269232-c94f9980-373a-11eb-9d3d-2e1113dfd27f.png)

Reply:
Yes! That would follow the pattern for our [volunteer form](https://covidtracking.com/contact/volunteer). Sorry that was unclear, thanks again!

Reply:
No worries! Happy to help out.